test/SlowTests/Issues/RavenDB-17624.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Client;
+using Orders;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Session;
+using Raven.Client.Documents.Subscriptions;
+using Raven.Client.Exceptions.Documents.Subscriptions;
+using Xunit;
+using Xunit.Abstractions;
+
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_17624 : RavenTestBase
+    {
+        public RavenDB_17624(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task ForbidOpeningMoreThenOneSessionPerSubscriptionBatch()
```

Comment:
I do not understand why this test is so complex, has some many console writelines, etc

Reply:
It was originally copied from the original scenario of the user